src/db/services/endpoints.js

```diff
@@ -7,6 +7,14 @@ exports.createEndpoint = body => {
   return endpoint.save()
 }
 
+exports.readEndpoint = endpointId => {
+  return EndpointModel.findById(endpointId)
+}
+
+exports.readEndpoints = () => {
+  return EndpointModel.find({})
+}
```

Comment:
@bradsawadye I wouldn't hard code in an empty object, allowing some specification would be useful

```suggestion
exports.readEndpoints = (queryParams) => {
  return EndpointModel.find(queryParams)
}
```

Reply:
addressed in https://github.com/jembi/openhim-mediator-mapping/pull/52/commits/af01ffe80fbe3b3e9e09d17f51106e1cf70d8f3a